### ISSUE TITLE
Remove Button variant - outline white disabled

### ISSUE THREAD

**Issue by laugerud** [bring/hedwig#1647]
This is only regarding the **disabled** version of the outline white button.

- This fails WCAG contrast
- Also nobody uses it so why bother fixing it
